packages/wd-button/src/WDButton.js

```diff
@@ -21,6 +22,179 @@ WDButton.STATE = {
    * @default example
    */
   id: Config.string().value('example'),
+
+  /**
+   * @type {?boolean}
+   * @default false
+   */
+  disabled: Config.bool().value(false),
+
+  /**
+   * @type {?(string|undefined)}
+   * @default undefined
+   */
+  elementClasses: Config.string(),
+
+  /**
+   * @type {?string}
+   * @default 'primary'
+   */
+  style: Config.oneOf(['primary', 'outline', 'ghost', 'danger', 'destructive', 'float']).value('primary'),
+
+  /**
+   * @type {?string}
+   * @default 'accent'
+   */
+  color: Config.oneOf(['accent', 'black', 'green', 'blue-light', 'blue', 'violet', 'red', 'orange', 'yellow']).value('accent'),
+
+  /**
+   * @type {?string}
+   * @default 'm'
+   */
+  size: Config.oneOf(['xs', 's', 'm', 'l', 'xl', 'xxl']).value('s'),
+
+  /**
+   * @type {?string|undefined}
+   * @default undefined
+   */
+  width: Config.string(),
+
+  /**
+   * @type {?boolean}
+   * @default false
+   */
+  hardWidth: Config.bool().value(false),
+
+  /**
+   * @type {?string}
+   * @default undefined
+   */
+  weight: Config.oneOf(['lightweight', 'middleweight', 'heavyweight']),
```

Comment:
You can remove that. we'll use the modifiers of `typography` and pass them in `elementClasses` parameter